### PR TITLE
Fix Linux compilation errors

### PR DESCRIPTION
**Pull request by digitcrusher**
Fixes:
- Fix a conditional operator in xrDebug.cpp
- Add ListWnd to build rules in xrUICore
- Fix an include in UIListWnd.cpp

### CODE REVIEW

File: src/xrCore/xrDebug.cpp [hunk 1]
```diff
@@ -712,7 +712,7 @@ void xrDebug::SetupExceptionHandler()
     else if (!strstr(commandLine, "-detailed_minidump"))
         minidumpFlags |= MiniDumpFilterMemory;
 #endif
-
```
Comment (Xottab-DUTY) [inline src/xrCore/xrDebug.cpp]: Please, remove those styling changes

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrCore/xrDebug.cpp [hunk 2]
```diff
@@ -939,6 +939,6 @@ void xrDebug::Initialize(pcstr commandLine)
 #ifdef DEBUG
     ShowErrorMessage = true;
 #else
-    ShowErrorMessage = commandLine ? strstr(commandLine, "-show_error_window") : false;
+    ShowErrorMessage = commandLine ? (bool)strstr(commandLine, "-show_error_window") : false;
```
Comment (Xottab-DUTY) [inline src/xrCore/xrDebug.cpp]: It's better like that:
```suggestion
    ShowErrorMessage = commandLine ? !!strstr(commandLine, "-show_error_window") : false;
```

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrCore/xrDebug.cpp (hunk 1 above)
Comment (digitcrusher) [reply src/xrCore/xrDebug.cpp]: Sorry, my editor automatically removes spaces at the end of lines.

File: src/xrCore/xrDebug.cpp [hunk 3]
```diff
@@ -712,7 +712,7 @@ void xrDebug::SetupExceptionHandler()
     else if (!strstr(commandLine, "-detailed_minidump"))
         minidumpFlags |= MiniDumpFilterMemory;
 #endif
-    
+
```
Comment (Xottab-DUTY) [inline src/xrCore/xrDebug.cpp]:
```suggestion
    
```

File: src/xrUICore/ListWnd/UIListWnd.cpp [hunk 4]
```diff
@@ -81,9 +81,9 @@ void CUIListWnd::InitListWnd(Fvector2 pos, Fvector2 size, float item_height)
 
     /*
         m_StaticActiveBackground.Init(ACTIVE_BACKGROUND,"hud\\default", 0,0,alNone);
-        m_StaticActiveBackground.SetTile(iFloor(m_iItemWidth/ACTIVE_BACKGROUND_WIDTH), 
+        m_StaticActiveBackground.SetTile(iFloor(m_iItemWidth/ACTIVE_BACKGROUND_WIDTH),
```
Comment (Xottab-DUTY) [inline src/xrUICore/ListWnd/UIListWnd.cpp]:
```suggestion
        m_StaticActiveBackground.SetTile(iFloor(m_iItemWidth/ACTIVE_BACKGROUND_WIDTH), 
```

File: src/xrUICore/ListWnd/UIListWnd.cpp [hunk 5]
```diff
@@ -81,9 +81,9 @@ void CUIListWnd::InitListWnd(Fvector2 pos, Fvector2 size, float item_height)
...
                                          iFloor(m_iItemHeight/ACTIVE_BACKGROUND_HEIGHT),
-                                         fmod(m_iItemWidth,float(ACTIVE_BACKGROUND_WIDTH)), 
+                                         fmod(m_iItemWidth,float(ACTIVE_BACKGROUND_WIDTH)),
```
Comment (Xottab-DUTY) [inline src/xrUICore/ListWnd/UIListWnd.cpp]:
```suggestion
                                         fmod(m_iItemWidth,float(ACTIVE_BACKGROUND_WIDTH)), 
```

File: src/xrUICore/ListWnd/UIListWnd.cpp [hunk 6]
```diff
@@ -107,9 +107,9 @@ void CUIListWnd::SetWidth(float width)
 {
     inherited::SetWidth(width);
     /*
-        m_StaticActiveBackground.SetTile(iFloor(GetWidth()/ACTIVE_BACKGROUND_WIDTH), 
+        m_StaticActiveBackground.SetTile(iFloor(GetWidth()/ACTIVE_BACKGROUND_WIDTH),
```
Comment (Xottab-DUTY) [inline src/xrUICore/ListWnd/UIListWnd.cpp]:
```suggestion
        m_StaticActiveBackground.SetTile(iFloor(GetWidth()/ACTIVE_BACKGROUND_WIDTH), 
```

File: src/xrUICore/ListWnd/UIListWnd.cpp [hunk 7]
```diff
@@ -107,9 +107,9 @@ void CUIListWnd::SetWidth(float width)
...
                                          iFloor(m_iItemHeight/ACTIVE_BACKGROUND_HEIGHT),
-                                         fmod(GetWidth(),float(ACTIVE_BACKGROUND_WIDTH)), 
+                                         fmod(GetWidth(),float(ACTIVE_BACKGROUND_WIDTH)),
```
Comment (Xottab-DUTY) [inline src/xrUICore/ListWnd/UIListWnd.cpp]:
```suggestion
                                         fmod(GetWidth(),float(ACTIVE_BACKGROUND_WIDTH)), 
```

File: src/xrUICore/ListWnd/UIListWnd.cpp [hunk 8]
```diff
@@ -220,7 +220,7 @@ void CUIListWnd::UpdateList()
 
     LIST_ITEM_LIST_it it = m_ItemList.begin();
 
-    //спрятать все элементы до участка 
+    //спрятать все элементы до участка
```
Comment (Xottab-DUTY) [inline src/xrUICore/ListWnd/UIListWnd.cpp]:
```suggestion
    //спрятать все элементы до участка 
```

**Comment by Xottab-DUTY** [comment]
Thank you!

**Pull request merged**